.travis.yml

```diff
@@ -5,23 +5,34 @@ os:
   - macos
   - linux
 elixir:
-  - 1.6
-  - 1.7
-  - 1.8
-  - 1.9
+  - '1.6'
+  - '1.7'
+  - '1.8'
+  - '1.9'
+  - '1.10'
 otp_release:
-  - 19.3
-  - 20.3
-  - 21.0
+  - '19.3'
+  - '20.3'
+  - '21.0'
+  - '22.2'
 matrix:
   exclude:
     - os: windows
-    - elixir: 1.8
-      otp_release: 19.3
-    - elixir: 1.9
-      otp_release: 19.3
+    - elixir: '1.6'
+      otp_release: '22.2'
+    - elixir: '1.7'
+      otp_release: '22.2'
+    - elixir: '1.8'
+      otp_release: '19.3'
+    - elixir: '1.9'
+      otp_release: '19.3'
+    - elixir: '1.10'
+      otp_release: '19.3'
+    - elixir: '1.10'
+      otp_release: '20.3'
 script:
   - if [[ "$TRAVIS_OS_NAME" = 'windows' ]]; then ./bin/build_win32; else ./bin/build; fi
+  - if [[ "$TRAVIS_ELIXIR_VERSION" = 1.10* ]]; then mix format --check-formatted; fi
```

Comment:
Because `mix format` uses different rules on different Elixir versions, here we just use the latest Elixir version, which is `1.10.1` currently.

It may not be the best choice to hardcode a version number here, but I didn't find any other way better.